model/qti/metadata/importer/MetaMetadataImportMapper.php

```diff
@@ -98,6 +100,11 @@ private function matchProperty(array &$metaMetadataProperty, array $classPropert
 
     private function isSynced(Property $classProperty, array &$metaMetadataProperty): bool
     {
+        $rangeClass = $classProperty->getRange();
+        $listClass = new \core_kernel_classes_Class(self::RDF_LIST);
```

Comment:
Keep it in import section. Maybe you could make it alias
```suggestion
        $listClass = new core_kernel_classes_Class(self::RDF_LIST);
```